docs/modules/ROOT/pages/advanced-how-tos/using-trusted-artifacts.adoc

```diff
@@ -0,0 +1,38 @@
+= Using Trusted Artifacts
+
+Konflux relies on Trusted Artifacts to securely allow sharing of data between Tasks without
+compromising the build process. See
+link:https://konflux-ci.dev/architecture/ADR/0036-trusted-artifacts.html[ADR-36] for more
+information.
+
+Today, when a component is onboarded to Konflux, the provided build Pipeline uses Trusted Artifacts.
```

Comment:
The word "create" is used instead of "onboard" in https://konflux-ci.dev/docs/how-tos/creating/